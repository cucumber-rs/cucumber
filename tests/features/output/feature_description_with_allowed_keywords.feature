Feature: output
  Markdown
  * item 1
  * item 2
  But is also supported
  And even
  Given is supported too

  Scenario: output
    Given foo is 0
    When foo is 1
    Then foo is 2
