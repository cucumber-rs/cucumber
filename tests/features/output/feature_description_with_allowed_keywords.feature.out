Started
Feature(Feature { keyword: "Feature", name: "output", description: Some("Markdown\n* item 1\n* item 2\nBut is also supported\nAnd even\nGiven is supported too"), background: None, scenarios: [Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Started)
Feature(Feature { keyword: "Feature", name: "output", description: Some("Markdown\n* item 1\n* item 2\nBut is also supported\nAnd even\nGiven is supported too"), background: None, scenarios: [Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }, Started))
Feature(Feature { keyword: "Feature", name: "output", description: Some("Markdown\n* item 1\n* item 2\nBut is also supported\nAnd even\nGiven is supported too"), background: None, scenarios: [Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }, Step(Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Started)))
Feature(Feature { keyword: "Feature", name: "output", description: Some("Markdown\n* item 1\n* item 2\nBut is also supported\nAnd even\nGiven is supported too"), background: None, scenarios: [Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }, Step(Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Passed(CaptureLocations(Locations([Some(0), Some(8), Some(7), Some(8)]))))))
Feature(Feature { keyword: "Feature", name: "output", description: Some("Markdown\n* item 1\n* item 2\nBut is also supported\nAnd even\nGiven is supported too"), background: None, scenarios: [Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }, Step(Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Started)))
Feature(Feature { keyword: "Feature", name: "output", description: Some("Markdown\n* item 1\n* item 2\nBut is also supported\nAnd even\nGiven is supported too"), background: None, scenarios: [Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }, Step(Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Passed(CaptureLocations(Locations([Some(0), Some(8), Some(7), Some(8)]))))))
Feature(Feature { keyword: "Feature", name: "output", description: Some("Markdown\n* item 1\n* item 2\nBut is also supported\nAnd even\nGiven is supported too"), background: None, scenarios: [Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }, Step(Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }, Started)))
Feature(Feature { keyword: "Feature", name: "output", description: Some("Markdown\n* item 1\n* item 2\nBut is also supported\nAnd even\nGiven is supported too"), background: None, scenarios: [Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }, Step(Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }, Passed(CaptureLocations(Locations([Some(0), Some(8), Some(7), Some(8)]))))))
Feature(Feature { keyword: "Feature", name: "output", description: Some("Markdown\n* item 1\n* item 2\nBut is also supported\nAnd even\nGiven is supported too"), background: None, scenarios: [Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Scenario(Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }, Finished))
Feature(Feature { keyword: "Feature", name: "output", description: Some("Markdown\n* item 1\n* item 2\nBut is also supported\nAnd even\nGiven is supported too"), background: None, scenarios: [Scenario { keyword: "Scenario", name: "output", description: None, steps: [Step { keyword: "Given ", ty: Given, value: "foo is 0", docstring: None, table: None, position: LineCol { line: 10 } }, Step { keyword: "When ", ty: When, value: "foo is 1", docstring: None, table: None, position: LineCol { line: 11 } }, Step { keyword: "Then ", ty: Then, value: "foo is 2", docstring: None, table: None, position: LineCol { line: 12 } }], examples: [], tags: [], position: LineCol { line: 9 } }], rules: [], tags: [], position: LineCol { line: 1 }, }, Finished)
Finished
